### PR TITLE
nodepick fix for BB env

### PR DESCRIPTION
**Pull request by dorinhogea**
fix R7 remsql to run on BB dev

### CODE REVIEW

File: db/fdb_boots.c [hunk 1]
```diff
@@ -355,13 +355,15 @@ static int _discover_remote_db_nodes(const char *dbname, const char *class,
...
+    const char *comdb2dbname = "comdb2db";
 
     /* NOTE: test is dev */
     if (strncasecmp(class, "test", 4) == 0) {
         class = "dev";
+        comdb2dbname = "comdb3db";
     }
 
-    rc = cdb2_open(&db, "comdb2db", "default", 0);
+    rc = cdb2_open(&db, comdb2dbname, "default", 0);
```
Comment (mohitkhullar) [inline db/fdb_boots.c]: This won't work on beta and alpha.
instead of default, you can put class with its default as prod, as comdb3db exists only on dev.

Comment (dorinhogea) [reply db/fdb_boots.c]: Why is "default" not working on alpha and beta CDB2 machines?

Comment (dorinhogea) [reply db/fdb_boots.c]: I would have preferred to have name mapping in cdb2api, this way server could say comdb2db and talk to whatever db is behind (comdb3db in BB dev case).

**Comment by dorinhogea** [comment]
This would work, until we decide to better encapsulate stage details inside a BB plugin.

**Comment by mohitkhullar** [comment]
this should work

**Comment by dorinhogea** [comment]
Thanks for review!

**Pull request merged**